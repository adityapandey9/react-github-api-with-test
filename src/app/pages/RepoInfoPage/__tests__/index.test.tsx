import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { RepoInfoPage } from '..';

const shallowRenderer = createRenderer();

describe('<RepoInfoPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<RepoInfoPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
