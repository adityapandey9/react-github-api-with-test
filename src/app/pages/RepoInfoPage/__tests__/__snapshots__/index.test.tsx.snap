// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RepoInfoPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      User Name Repo Page
    </title>
    <meta
      content="Search using Github Username"
      name="description"
    />
  </Helmet>
  <NavBar />
  <PageWrapper>
    <Features />
  </PageWrapper>
</React.Fragment>
`;
